### DISCUSSION TITLE
Am I installing course wrong? First 3 cargo commands run fine, 4th gives "--path must point to directory" error

### DISCUSSION THREAD

**Discussion by RagManX** [google/comprehensive-rust#666] [Q&A] [answered]
I really want to try out Comprehensive Rust, but I'm failing on just trying to install it and start running the local mdbook server. I've installed Rust on my Ubuntu VM and checked that the rustc and cargo executables work. Running the cargo install commands for mdbook, mdbook-svgbob, and mdbook-i18n-helpers all seemed to my inexperienced eye to run correctly and to completion, ending with "Installed package \<package name\>" However, the fourth command, copied from the github README.md document and pasted in my xterm, gives me an error and I have no idea how to diagnose/fix it and proceed to the mdbook test step:

[$]> cargo install --path mdbook-exerciser
error: `/home/raw/mdbook-exerciser` does not contain a Cargo.toml file. --path must point to a directory containing a Cargo.toml file.

I am running this as a non-root user, and my shell is zsh. Searching the discussions here doesn't turn up anyone else posting about the --path error I'm having. Not sure any other information I need to provide, but will gladly share anything else if needed. Could someone be so kind as to help me troubleshoot this so I can dig into learning Rust?

**Comment by mgeisler** [comment, marked as answer]
Hi @RagManX, thanks for trying out the course!

You have to run the command from the root of the checkout. You probably just have to `cd comprehensive-rust` before `cargo install --path mdbook-exerciser`.

I hope that helps!

**Comment by RagManX** [comment]
Sure enough, that was I wasn't in the comprehensive-rust directory when running through the build steps. Thanks for pointing out what I was missing!

**Reply by mgeisler** [reply]
Perfect, I'm glad I could help!
